pkg/plugins/generic/generic_plugin.go

```diff
@@ -269,7 +271,11 @@ func needDriverCheckVdpaType(state *sriovnetworkv1.SriovNetworkNodeState, driver
 // editKernelArg Tries to add the kernel args via ostree or grubby.
 func editKernelArg(helper helper.HostHelpersInterface, mode, karg string) error {
 	log.Log.Info("generic plugin editKernelArg()", "mode", mode, "karg", karg)
-	_, _, err := helper.RunCommand("/bin/sh", scriptsPath, mode, karg)
+	script := daemonScriptsPath
+	if _, err := os.Stat(consts.Host); errors.Is(err, os.ErrNotExist) {
+		script = systemdScriptsPath
+	}
+	_, _, err := helper.RunCommand("/bin/bash", script, mode, karg)
```

Comment:
can we stay with `sh` here as before? or swtich the other calls to `bash` but please lets be consistent

Reply:
an updated scripts doesn't work with `sh` beacause it uses `bash`. do we want ti switch to `bash` everywhere?